build/azDevops/azure/azure-pipelines-javaspring-k8s.yml

```diff
@@ -92,6 +93,7 @@ variables:
   # BUILD ARTIFACTS across stages
   build_artifact_deploy_path: $(Agent.BuildDirectory)/s/$(self_repo)/deploy/k8s/app
   build_artifact_deploy_name: $(self_generic_name)
+  build_scripts_directory: "$(Agent.BuildDirectory/s/$(self_pipeline_repo)/scripts"
```

Comment:
This is relating to another branch, best move it to the other branch before merging.